app/assets/stylesheets/show.scss

```diff
@@ -17,6 +16,26 @@
   }
 }
 
+.embed-container {
+  position: relative;
```

Comment:
Properties should be ordered height, height, margin, overflow, padding-bottom, padding-top, position, width